doc/concepts/variables.rst

```diff
@@ -48,13 +48,13 @@ Dune supports the following variables:
    are the file extensions used for various artifacts.
 - ``ext_plugin`` is ``.cmxs`` if ``natdynlink`` is supported and
   ``.cma`` otherwise.
-- ``ocaml-config:v`` is for every variable ``v`` in the output of
-  ``ocamlc -config``. Note that Dune processes the output
-  of ``ocamlc -config`` in order to make it a bit more stable across
-  versions, so the exact set of variables accessible this way might
-  not be exactly the same as what you can see in the output of
-  ``ocamlc -config``. In particular, variables added in new OCaml versions
-  need to be registered in Dune before they can be used.
+- ``ocaml-config:v`` expands for every configuration variable ``v`` to the
```

Comment:
```suggestion
- ``ocaml-config:v`` expands every configuration variable ``v`` to the
```

Reply:
At this point given that the variables have to be known to Dune anyway and we can't just say "anything in `ocaml -config`", maybe it would make sense to just document those that exist?

To me the paragraph seems to spend a lot of time stating what `v` does before actually saying that it isn't really all that and there is only a limited set of values for `v` but not which values are actually available.